### ISSUE TITLE
RefreshDatabaseTrait repopulates the database on every new TestCase

### ISSUE THREAD

**Issue by Philostastically** [theofidry/AliceBundle#17]
Our tests have been running slowly for a while now and I finally got around to looking into it, and I think its because the variable `$dbPopulated` in `RefreshDatabaseTrait` is always false. Or at least is false on each new class it's instantiated on to.  This means it repopulates the database each time it moves to a new TestCase which uses `RefreshDatabaseTrait`. 

If I replace RefreshDatabaseTrait with this: 
```
trait TestTrait
{
    private static int $x = 0;

    protected static function bootKernel(array $options = [])
    {
        $kernel = parent::bootKernel($options);

        if (!static::$x) {
            echo "x = ".self::$x.PHP_EOL;
            static::$x++;
        }
        return $kernel;
    }
}
```

It just outputs :
```
x = 0
x = 0
x = 0
x = 0
```

I'm using PHP 8.1.3, and I think it's because [calling a static element on a trait was deprecated](https://www.php.net/manual/en/migration81.deprecated.php) in 8.1.0.

**Comment by theofidry** [comment]
This is a bit annoying indeed...

**Comment by Cafeine42** [comment]
We had this problem, worked around with an abstract class that holds the trait, used by all others tests class.

**Comment by louismariegaborit** [comment]
@theofidry What is the solution you recommend to work around this problem? It's a very annoying problem because we lose a lot of time at the beginning of each class.

**Comment by TravisMarkvh** [comment]
I missed this issue but we ran into the same problem.
Created this PR to fix the issue
https://github.com/theofidry/AliceBundle/pull/42

**Comment by theofidry** [comment]
Closed by #43, which is included in the latest tag.
